src/sentry/api/endpoints/organization_projectoptions.py

```diff
@@ -0,0 +1,29 @@
+from rest_framework.request import Request
+from rest_framework.response import Response
+
+from sentry.api.base import pending_silo_endpoint
+from sentry.api.bases.organization import Organization, OrganizationEndpoint, OrganizationPermission
+from sentry.api.bases.project import Project
+from sentry.api.serializers import serialize
+from sentry.api.serializers.models import ProjectOptionsSerializer
+
+
+@pending_silo_endpoint
+class OrganizationProjectOptionsEndpoint(OrganizationEndpoint):
```

Comment:
should this be a private or public API?

Reply:
🤷 is the existing projects api public? may as well just keep it as is

Reply:
public is good

Reply:
we're making it private since the use cases of this will very likely only be internal